nekoyume/Assets/_Scripts/UI/Widget/Popup/BuffBonusPopup.cs

```diff
@@ -27,6 +29,21 @@ public class BuffBonusPopup : PopupWidget
 
         private System.Action _onAttract;
 
+        [SerializeField]
+        private Button buffListButton = null;
+
+        [SerializeField]
+        private GameObject buffListView = null;
+
+        [SerializeField]
+        private Transform cellContainer;
+
+        [SerializeField]
+        private GameObject titlePrefab;
+
+        [SerializeField]
+        private GameObject buffPrefab;
```

Comment:
```suggestion
        private BuffBonusBuffCell buffPrefab;
```

If you change type of this prefab object, you would not need `GetComponent` in `initializeBuffListView()`.